### PR TITLE
[doc] add seed peers for devnet

### PR DESCRIPTION
**Pull request by sherry-x**

### CODE REVIEW

File: developer-docs-site/docs/tutorials/run-a-fullnode.md [hunk 1]
```diff
@@ -94,6 +94,24 @@ During the initial synchronization of your FullNode, there may be a lot of data
   du -cs -BM /opt/aptos/data
   ```
 
+#### Add upstream seed peers
+Devnet validator fullnode will only accept maximum of 1000 connections, if our network is experiencing high volume, your fullnode might not able to connect. If this happens, you can set `seeds` in the fullnode configuration file to add new upstream peers for connection. We prepared some extra fullnode address for you to use:
```
Comment (rustielin) [inline developer-docs-site/docs/tutorials/run-a-fullnode.md:98]: Can you include an example of what happens when your node is unable to connect? e.g. metrics for upstream peers, logs etc

Comment (gregnazario) [reply developer-docs-site/docs/tutorials/run-a-fullnode.md:98]: Yeah, putting that the metric of `diem_connections` would be 0, would be a good show of that it's possible you are not getting any connections.

Additionally, state sync won't increase it's local state

**Review by gregnazario — APPROVED** [review]

**Pull request merged**